samples/msgext-ai-doc-compliance-checker/README.md

```diff
@@ -0,0 +1,130 @@
+
+**Compliance Checker sample app:** [Manifest](/samples/msgext-ai-doc-compliance-checker/demo-manifest/AIComplianceChecker.zip)
+
+## Prerequisites
+
+- [Node.js 18.x](https://nodejs.org/download/release/v18.18.2/)
+- [Visual Studio Code](https://code.visualstudio.com/)
+- [Blob Storage](https://learn.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-portal)
+- [Teams Toolkit](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension)
+- You will need a Microsoft work or school account with [permissions to upload custom Teams applications](https://learn.microsoft.com/microsoftteams/platform/concepts/build-and-test/prepare-your-o365-tenant#enable-custom-teams-apps-and-turn-on-custom-app-uploading). The account will also need a Microsoft Copilot for Microsoft 365 license to use the extension in Copilot.
+- [Azure Open AI](https://learn.microsoft.com/en-us/azure/ai-services/openai/quickstart?tabs=command-line&pivots=programming-language-studio)
+
+## Setup and use the sample
+
+1) **Create Azure Blob Storage:**
+Refer to the Create Blob Storage documentation [Blob Storage](https://learn.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-portal) to create a blob storage in Azure.
+
+1) **Add the files to be checked for compliance:**
+Upload the policy guideline document (keep the file-name handy for adding into the .localConfigs file) and also the proposal documents to compare against the guideline.
+
+1) Clone the repository
+
+    ```bash
+    git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+    ```
+1) Navigate to the `samples/msgext-ai-doc-compliance-checker` folder and open with Visual Studio Code.
+
+1) Navigate to the `samples/msgext-ai-doc-compliance-checker/.localConfigs` directory and update the values below.
+
+   ```txt
+      END_POINT={{Azure_End_Point}}
+      API_KEY={{Azure_Api_Key}}
+      DEPLOYMENT_ID={{Azure_Deployment_Id}}
+      AZURE_STORAGE_CONNECTION_STRING={{Azure_Storage_Connection_String}}
+      AZURE_CONTAINER_NAME={{Azure_Container_Name}}
+      CHECKLIST_NAME={{Name of the Policy Guideline document uploaded in Azure Blob}}
+   ```
+
+## Run the app (Using Teams Toolkit for Visual Studio Code)
+
+The simplest way to run this sample in Teams is to use Teams Toolkit for Visual Studio Code.
+
+1. Ensure you have downloaded and installed [Visual Studio Code](https://code.visualstudio.com/docs/setup/setup-overview)
+1. Install the [Teams Toolkit extension](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension)
+1. Select **File > Open Folder** in VS Code and choose this samples directory from the repo
+1. Using the extension, sign in with your Microsoft 365 account where you have permissions to upload custom apps
+1. Select **Debug > Start Debugging** or **F5** to run the app in a Teams web client.
```

Comment:
Step is missing to First create Azure Open AI service and BLob storage before running this step.

Reply:
Added Fixed